code/game/objects/items/weaponry.dm

```diff
@@ -20,7 +20,7 @@
 
 /obj/item/banhammer/attack(mob/M, mob/user)
 	if(user.zone_selected == BODY_ZONE_HEAD)
-		M.visible_message("<span class='danger'>[user] are stroking the head of [M] with a bangammer</span>", "<span class='userdanger'>[user] are stroking the head with a bangammer</span>", "you hear a bangammer stroking a head");
+		M.visible_message("<span class='danger'>[user] is stroking the head of [M] with a banhammer</span>", "<span class='userdanger'>[user] is stroking the head with a banhammer</span>", "you hear a banhammer stroking a head");

```

Comment:
Use `[user] [user.p_are()] stroking`. It'll come out to "is" most of the time but remain "are" in the narrow cases where doing so is appropriate.

Reply:
Actually I'm not entirely sure about this, I'll take a deeper look at some point.